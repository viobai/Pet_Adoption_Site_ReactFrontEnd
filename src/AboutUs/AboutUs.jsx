import React from 'react';
import style from './AboutUs.module.css';

const AboutUs = () => {
    return (
        <div className={style.aboutUsPageContainer}>
            <img className={style.imgAtBack} alt="Pet Owner Washing His Dog" src="https://github.com/viobai/seainfo6150-final-project-webapp/blob/master/src/images/aboutUsImg.png?raw=true" />
             <div className={style.column}>
                    <div className={style.card}>
                        <h2 className={style.subTitle}>Mission Statement</h2>
                        <p>Seattle Humane promotes the human-animal bond by saving and serving pets in need, 
                            regardless of age, ability, circumstance or geography.</p>
                    </div>
                    <div className={style.card}>
                        <h2 className={style.subTitle}>Volunteer Opportunities</h2>
                        <p><span className={style.update}>
                            *** Covid-19 Updates: due to the current pandemic, we are not taking in any new 
                            volunteers. Stay safe! ***
                        </span></p>
                        <p>
                            We welcome volunteers in all ages and regardless of experience! Feel free to 
                            shoot us an email, give a call or even visit us in person to request for more 
                            details.
                        </p>
                    </div>
                    <div className={style.card}>
                        <h2 className={style.subTitle}>Financials</h2>
                        <p>
                            <span className={style.orgName}>Rescue Project: Humane</span> is nonprofit 
                            organization. We value transparency, our financial statements are made available to 
                            the public by the end of each tax season.
                        </p>
                        <p>Our tax-ID is XX-XXXXXXX and donations are tax-deductible to the extent allowed by law.</p>
                    </div>
                </div>
        </div>
               
    )
}

export default AboutUs
