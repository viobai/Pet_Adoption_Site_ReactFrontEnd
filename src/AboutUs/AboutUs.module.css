.card {
    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
    margin: 8px;
    background-color: #fdf8f6;
  }

.column {
  width: 80%;
  margin-left: 10%;
  margin-right: 10%;
  
}

.aboutUsContainer {
  background: #d9ccc6;
  padding: 3rem 0 1rem 0;
}

.container {
    padding: 10px 15px;
    margin-bottom: 3rem;
  }
  
.container::after, .row::after {
    content: "";
    clear: both;
    display: table;
}

.title {
    color: rgb(211, 69, 50);
}