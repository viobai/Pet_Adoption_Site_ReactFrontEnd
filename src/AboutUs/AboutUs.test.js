import React from "react";
import AboutUs from "./AboutUs";

describe("AboutUs tests", () => {
    it("renders correctly", () => {  
        const { container } = render(<AboutUs/>);
        expect(container).toMatchSnapshot();
    });
});