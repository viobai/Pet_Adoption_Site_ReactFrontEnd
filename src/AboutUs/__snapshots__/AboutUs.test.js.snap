// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs tests renders correctly 1`] = `
<div>
  <div
    class="aboutUsPageContainer"
  >
    <img
      alt="Pet Owner Washing His Dog"
      class="imgAtBack"
      src="https://github.com/viobai/seainfo6150-final-project-webapp/blob/master/src/images/aboutUsImg.png?raw=true"
    />
    <div
      class="column"
    >
      <div
        class="card"
      >
        <h2
          class="subTitle"
        >
          Mission Statement
        </h2>
        <p>
          Seattle Humane promotes the human-animal bond by saving and serving pets in need, regardless of age, ability, circumstance or geography.
        </p>
      </div>
      <div
        class="card"
      >
        <h2
          class="subTitle"
        >
          Volunteer Opportunities
        </h2>
        <p>
          <span
            class="update"
          >
            *** Covid-19 Updates: due to the current pandemic, we are not taking in any new volunteers. Stay safe! ***
          </span>
        </p>
        <p>
          We welcome volunteers in all ages and regardless of experience! Feel free to shoot us an email, give a call or even visit us in person to request for more details.
        </p>
      </div>
      <div
        class="card"
      >
        <h2
          class="subTitle"
        >
          Financials
        </h2>
        <p>
          <span
            class="orgName"
          >
            Rescue Project: Humane
          </span>
           is nonprofit organization. We value transparency, our financial statements are made available to the public by the end of each tax season.
        </p>
        <p>
          Our tax-ID is XX-XXXXXXX and donations are tax-deductible to the extent allowed by law.
        </p>
      </div>
    </div>
  </div>
</div>
`;
